lib/rspec/core/formatters.rb

```diff
@@ -212,6 +213,8 @@ def built_in_formatter(key)
         JsonFormatter
       when 'bisect-drb'
         BisectDRbFormatter
+      when 'c'
+        CFormatter
```

Comment:
`CFormatter` isn't a good name for this, we're not C, a better name might be `FileReporter` or `MinimalReporter`

Reply:
Hehe… naming things… :scream:

Vim display this information in it's "[quickfix window](http://vimdoc.sourceforge.net/htmldoc/quickfix.html)", EMACS in a "[compilation buffer](https://www.gnu.org/software/emacs/manual/html_node/emacs/Compilation-Mode.html)".  I wanted the name to be editor agnostic but also confess that choosing "c" when doing "ruby" is weird.

`MinimalReporter` seems a better name, I'll add a commit to fix the names, thanks for the suggestion!

Reply:
@JonRowe just saw you wrote "Reporter" and not "Formatter".  I don't see any other file with a name including "reporter", can you please tell me if it was a typo or on purpose?

Reply:
Er yes, I meant `Formatter`

Reply:
Okay, so you meant and I read the same "Formatter" thing, so I committed the right thing, just wanted to be sure :smile:.

I will resolve this conversation then.